src/main/java/net/pms/formats/SAMI.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
@Nadahar Reading https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/d948893dbd963a268e2fc093b3d09b5037560974
and 2 others patch from 2012 make come to **SMIL** support by FFMpeg, SMI subtitles use a part of this code.

So why not add this subtitle possibility ? Not necessary in this PR, just to put the subject on the table.